src/docs/sphinx/BuildingAscent.rst

```diff
@@ -471,20 +486,14 @@ Building Ascent in a Docker Container
 Under ``src/examples/docker/master/ubuntu`` there is an example ``Dockerfile`` which can be used to create an ubuntu-based docker image with a build of the Ascent github master branch. There is also a script that demonstrates how to build a Docker image from the Dockerfile (``example_build.sh``) and a script that runs this image in a Docker container (``example_run.sh``). The Ascent repo is cloned into the image's file system at ``/ascent``, the build directory is ``/ascent/build-debug``, and the install directory is ``/ascent/install-debug``.
 
 
-.. _building_manually:
-
-Building Ascent Dependencies Manually
--------------------------------------
-
-In some environments, a spack build of Ascent's dependencies can fail or a user may prefer to build the dependencies manually.
+.. _building_extras:
 
-Here is a `script <https://github.com/Alpine-DAV/ascent/blob/develop/scripts/build_ascent/build_ascent.sh>`_ that demonstrates how to build Ascent and its main dependencies:
 
-.. literalinclude:: ../../../scripts/build_ascent/build_ascent.sh
-   :language: bash
+Special Case Builds
+------------------------
 
-This script supports building on linux, macOS, and windows.
-It supports building without device support, and with CUDA or ROCm/HIP.
+BabelFlow Support
+~~~~~~~~~~~~~~~~~
```

Comment:
for some reason this title is not being rendered for me. 
![no_babelflow](https://github.com/user-attachments/assets/2aeb29b3-6ce2-4e2a-98b5-72fa00d7d551)